generator/test_case_generator.go

```diff
@@ -181,8 +181,9 @@ var testTypeToTestConfig = map[string][]testConfig{
 	},
 	"eks_daemon": {
 		{
-			testDir: "./test/metric_value_benchmark",
-			targets: map[string]map[string]struct{}{"arc": {"amd64": {}}},
+			testDir:      "./test/metric_value_benchmark",
+			targets:      map[string]map[string]struct{}{"arc": {"amd64": {}}},
+			instanceType: "g4dn.xlarge",
```

Comment:
How can we make this test suite more flexible to support other instance types? We are going to eventually add e2e tests for Trainium, Inferentia and ARM instance types as well. I think this approach would make it harder to add more instance types down the road.

Reply:
I think we can eventually add the extensibility of this e2e for Trainium, Inferentia and ARM instance types as a follow up. But as of right now having this should be good, and in fact the e2e test uses the instance type given in the terraform script, so this isn't really needed...